### DISCUSSION TITLE
Manage Assests in a proper way in declarative local mode is not correct comared in management portal

### DISCUSSION THREAD

**Discussion by epaphrasmakoko** [openappsec/openappsec#241] [Feature Requests]
In Declarative Mode. When adding Asset through local_policy.yaml through Specific Rule there is an issue
1st. Because when you use fully managed portal and add an asset, the asset will be seen in Asset tab also in main dashboard
2nd. Also when adding an asset in full managed portal one can add alot of features like name of asset, tags, all URLs that are based on that particular asset eg an asset can have multiple URLs to be accessed but this is not in declarative mode
In delcarative mode when you add an asset the name of the url becomes the name of the asset

3rd. since the asset is added form declarative mode this means it can not be seen in asset tab in visibility mode in portal and this lead to when downloading the fully policy form the portal the policy will differ from that is inside the server policy.json

How do i add multiple urls in the asset in delcrative mode as it is in fully managed mode also names of assets

inshort how do we add the assets in a proper way just like in web management portal???

**Comment by orianelou** [comment]
Hi @epaphrasmakoko, 

Some asset features (such as name and tags, and multiple URLs per asset), are currently missing in our current version, but we'll have this fixed in our next declarative option released soon. 

Regarding assets not displayed in the web UI, declarative mode assets (specific rules), should be displayed, for us to investigate this please attach your local policy and tenant ID (you can also send it to info@openappsec.io) if you prefer.

Best, 

the open-appsec team

**Comment by epaphrasmakoko** [comment]
Okay that's okay but can you please update the Doc with the corresponding updates that you did recently 
The documentation is still the same for a long time
also about declarative mode its resolved and everything is working fine

**Comment by orianelou** [comment]
Hi @epaphrasmakoko, 

We have released a new local policy version which will allow you to manage in declarative mode in a similar way to web UI, let us know what you think! https://docs.openappsec.io/getting-started/start-with-linux/local-policy-file-v1beta2-beta
